client/a8c-for-agencies/sections/agency-tier/overview-content/tier-benefits.tsx

```diff
@@ -32,7 +32,9 @@ export default function TierBenefits( {
 } ) {
 	const dispatch = useDispatch();
 
-	const currentTier = ALL_TIERS.find( ( tier ) => tier.id === currentAgencyTierId );
+	const currentTier =
+		ALL_TIERS.find( ( tier ) => tier.id === currentAgencyTierId ) ??
+		ALL_TIERS.find( ( tier ) => tier.level === 0 );
```

Comment:
I see we have this duplicated in a few places. Maybe we can create a util function or something?

Reply:
Good one. Fixed it here: https://github.com/Automattic/wp-calypso/pull/106950/commits/8f9d5f738b07223874724173d27accccfd39553d